packages/ckeditor5-package-generator/lib/templates/ckeditor5-metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"plugins": [
+		{
+			"name": "<%= name.split('/')[1] %>",
+			"className": "<%= name.split('/')[1].replace(/ /g, '') %>",
+			"description": "Adds text to the editor.",
+			"path": "src/myplugin.js",
+			"docs": "features/text-alignment.html",
```

Comment:
The `docs` field could be skipped here, as it is optional and there is no docs for the `MyPlugin` plugin.